workspaces/linux-runtime/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+cargo-features = ["resolver"]
```

Comment:
Can you put an `authors`, `version`, and `description` field in this `Cargo.toml` file with authors "The Veracruz development team", version `0.3.0`, and an appropriate description.